### PR TITLE
[CLOSED] solana infrastructure migration docs - migrated to #1073

### PR DESCRIPTION
**Pull request by samgutentag**
documentation on existing and planned infrastructure changes with the migration to Solana

### CODE REVIEW

File: docs/solana-migration/migration-infrastructure.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+and the blockchain. These infrastructure changes will allow the Helium Network to effectively
+introduce new Networks as new Decentralized Network Protocol (DNP) subDAOs and more easily scale
+existing DNP subDAOs by removing existing bottlenecks. The following is an attempt to document the
+future state of the infrastructure post-migration.
+
+---
+
+## Price Oracle
+
+On the Helium L1, the Price Oracle iss the only existing Oracle on the Helium Network. It is
```
Comment (abhay) [inline docs/solana-migration/migration-infrastructure.mdx]:
```suggestion
On the Helium L1, the Price Oracle is the only existing Oracle on the Helium Network. It is
```

File: docs/solana-migration/migration-infrastructure.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+expansion of the Helium Network.
+
+### Config Service
+
+The Config Service will keep track of all the routing rules for all the devices connected to the
+Helium Network, aggregating from the LNSs into a database of routes that will propagate to each
+Packet Router. The Config Service will immediately update the Packet Routers whenever it receives an
+update.
+
+The Packet Verifier also uses the CONfig Service to fetch the list of OUIs, wallet addresses per
```
Comment (abhay) [inline docs/solana-migration/migration-infrastructure.mdx]:
```suggestion
The Packet Verifier also uses the Config Service to fetch the list of OUIs, wallet addresses per
```

**Comment by samgutentag** [comment]
mermaid diagrams will require #1055 

diagrams need some tweaking, not all formatting options are supported by docusaurus ☹️

👇 that merge adds mermaid diagrams 👇

**Comment by samgutentag** [comment]
closing in favor of #1073
merging existing changes to #1073

**Pull request closed without merging**